client/css/style.css

```diff
@@ -2090,7 +2090,7 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
    For example: user has quit ( ===> L   O   L <=== )  */
 
 #windows .header .topic,
-#chat .message .text,
+#chat .message .text span,
```

Comment:
BTW I believe that this change (as opposed to removing whitespaces in the template) would fix bugs 2 and 3 of https://github.com/thelounge/lounge/issues/1357.